### ISSUE TITLE
react beautiful dnd, ugly animation

### ISSUE THREAD

**Issue by Mulinatti** [Mulinatti/kanban-app#1]
The animation after moving tasks is very ugly, it doesn't move like the examples in the documentation. Maybe a CSS issue ?
It's my first time using this lib. I have no idea what it could be. I tried few things like change the CSS.

[Screencast from 14-06-2024 22:09:21.webm](https://github.com/Mulinatti/kanban-app/assets/79599663/534c2d8b-e1d5-46bf-9d47-7304489cea03)

**Comment by lodenrogue** [comment]
Are you talking about the snapping animation?

**Comment by Mulinatti** [comment]
Yes, when I release left click, it stops in a position, then it instantly changes. It happens when I change the items order.
